lib/normalize_title_filter.js

```diff
@@ -64,9 +64,14 @@ module.exports = function(hyper, req, next, options, specInfo) {
                             location: contentLocation
                         }
                     });*/
-                    hyper.log('warn/normalize', {
-                        msg: 'Normalized requested title from ' + rp.title + ' to ' + result
-                    });
+
+
+                    // 5% chance of logging to bound volume.
```

Comment:
The entire logging is temporary, and there is a comment to that effect on line 45.